hack/cluster.sh

```diff
@@ -3,7 +3,7 @@
 set -xe
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
-KIND_ARGS="${KIND_ARGS:--ic -ikv -i6 -mne}"
+KIND_ARGS="${KIND_ARGS:--ic -ikv -i6 -mne -nse}"
```

Comment:
do we want this on all lanes, or we want to add an additional lane, and test it there ? 

I'm inclined to say we want the latter.

Reply:
hmm.. regardless of the question whether we want to run the tests in a parallel lane, I don't think we should only enable it when we want to test it. 
It's a feature that is planned to be enabled by default, so I don't see a good reason why not to add it for all the cluster lanes.

Reply:
i also dont see why to add another lane tbh, we should not over test